DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyDropDown.cs

```diff
@@ -124,12 +138,20 @@ public void ApplySpawnIniCode(IniFile spawnIni)
                 case DropDownDataWriteMode.INDEX:
                     spawnIni.SetIntValue("Settings", spawnIniOption, SelectedIndex);
                     break;
-                default:
```

Comment:
your PR seems to break the default case. Move your `SPAWN_SPAWNMAP` right after the `INDEX` case and before the default case. Keep the `default` and `STRING` cases untouched